x-pack/plugins/enterprise_search/public/applications/app_search/index.tsx

```diff
@@ -16,13 +20,52 @@ export const AppSearch: React.FC = () => {
   const { enterpriseSearchUrl } = useContext(KibanaContext) as IKibanaContext;
 
   return (
-    <>
+    <Switch>
       <Route exact path="/">
-        {!enterpriseSearchUrl ? <Redirect to="/setup_guide" /> : <EngineOverview />}
+        {!enterpriseSearchUrl ? <Redirect to="/setup_guide" /> : <Redirect to="/engines" />}
```

Comment:
I think it would be good to go ahead and port over `app/javascript/app_search/utils/routePaths.ts` as referenced in previous comment. Already see where we could DRY out below

Reply:
Ya I've been lazy about route strings vs constants up until now - I'm leaning towards doing that in a separate PR though, since this one already has so many lines/moving parts